hazelcast/src/main/java/com/hazelcast/internal/jmx/ManagementService.java

```diff
@@ -101,10 +103,29 @@ public void destroy() {
         }
     }
 
-    public static void shutdownAll() {
+    public static void shutdownAll(List<HazelcastInstanceProxy> instances) {
+        MBeanServer mbs = ManagementFactory.getPlatformMBeanServer();
+        for (HazelcastInstanceProxy instance : instances) {
+            try {
```

Comment:
I think, it's better to change `shutdownAll(HazelcastInstanceImpl instance)` to  `shutdownAll(String instanceName)` and call it inside this for loop.
This way we can get rid of this nested for loop and avoid duplicate code.